backend/build.gradle

```diff
@@ -98,6 +111,7 @@ springBoot {
 test {
     // people sometimes export this to run local dev more easily: don't let it break the tests
     environment "SPRING_PROFILES_ACTIVE", ""
+    environment "OKTA_TESTING_DISABLEHTTPSCHECK", "true"
```

Comment:
Is this a Spring-managed property in the `okta.testing` namespace? If so, could/should it be in the `application-default.yaml` properties file?

Reply:
For some reason, applying it there doesn't work. The only way we could get this to apply properly is through env variables ¯\_(ツ)_/¯

Reply:
That's wild. I have some guesses as to how that could be the case, but they probably wouldn't change much. (Also I'm not 100% sure what this is doing, but it looks like a reasonable thing based on what it says.)

I guess it's possible that putting it in `application.yaml` would do it if I'm right, but it's not clear that there's a benefit to that.

Come to that, though, it would probably be good to have some comment on one of the places this is being done explaining what it does. 😁

Reply:
Will add a comment, but basically this makes the Okta client not complain/ throw exceptions when we use a non-https wiremock server as our org url!

Reply:
Added!